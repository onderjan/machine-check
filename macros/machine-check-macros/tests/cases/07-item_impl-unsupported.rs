#[machine_check_macros::machine_description]
mod machine {
    struct A {}

    default impl A {}

    unsafe impl A {}
}

fn main() {}
