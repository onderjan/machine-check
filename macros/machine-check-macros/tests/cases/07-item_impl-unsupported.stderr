error: machine-check: Defaultness not supported
 --> tests/cases/07-item_impl-unsupported.rs:5:5
  |
5 |     default impl A {}
  |     ^^^^^^^

error: machine-check: Unsafety not supported
 --> tests/cases/07-item_impl-unsupported.rs:7:5
  |
7 |     unsafe impl A {}
  |     ^^^^^^
